sdk/eventgrid/azure-messaging-eventgrid/README.md

```diff
@@ -42,6 +42,37 @@ az eventgrid domain create --location <location> --resource-group <your-resource
 ```
 
 ### Include the package
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on the latest GA version of the library.
+
+```xml
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-sdk-bom</artifactId>
+            <version>1.0.3</version>
```

Comment:
Since we are not including the version in any other READMEs, we should use the same template that we have in other places.